app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  after_create :posts_counter_update
```

Comment:
Great work using a helper method to invoke your private method. However, you're using after_create and we want to invoke this method on every update to the model so let's replace it with after_save

_Please apply this request to the other models._